src/benchmark.jl

```diff
@@ -244,17 +244,13 @@ function WorkPrecisionSet(prob,
   end
   for i in 1:N
     print_names && println(names[i])
-    if haskey(setups[i],:dts)
-      wps[i] = WorkPrecision(prob,setups[i][:alg],abstols,reltols,setups[i][:dts];
-                                 appxsol=appxsol,
-                                 error_estimate=error_estimate,
-                                 name=names[i],kwargs...,setups[i]...)
-    else
-      wps[i] = WorkPrecision(prob,setups[i][:alg],abstols,reltols;
-                                 appxsol=appxsol,
-                                 error_estimate=error_estimate,
-                                 name=names[i],kwargs...,setups[i]...)
-    end
+    _abstols = get(setups[i],:abstols,abstols)
+    _reltols = get(setups[i],:reltols,reltols)
+        _dts = get(setups[i],:dts,nothing)
```

Comment:
```suggestion
     dts = get(setups[i],:dts,nothing)
```

Reply:
It should be `_dts`?

Reply:
Oops, I meant to comment on the indentation. :P